src/utils/encoder.tsx

```diff
@@ -38,10 +39,38 @@ export function upload(
             const leftChannelData = new Float32Array(upsampledAudioBuffer.getChannelData(0));
             const rightChannelData = new Float32Array(upsampledAudioBuffer.getChannelData(1));
 
+            const applyHammingWindow = (data: Float32Array) => {
```

Comment:
I think this could be the reason for gh-101.

needs to be checked if removing this hamming window solves the bug with unstable volume